src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -73,6 +81,20 @@ export class HostsComponent extends ListWithDetails implements OnInit {
           );
         }
       },
+      {
+        name: this.actionLabels.EDIT,
+        permission: 'update',
+        icon: Icons.edit,
+        click: () => {
+          this.depCheckerService.checkOrchestratorOrModal(
+            this.actionLabels.EDIT,
+            this.i18n('Host'),
+            () => this.editAction()
+          );
+        },
+        disable: (selection: CdTableSelection) =>
+          !selection.hasSingleSelection || !selection.first().sources.orchestrator
```

Comment:
This can be removed as `depCheckerService` will open a modal telling you that you can't use edit if the orchestrator is not available. (Like it is done with the create action.)

Reply:
The `!selection.first().sources.orchestrator` is to ensure that the 'Edit' button is only available for hosts that are managed via Orchestrator. Labels are not supported for hosts that are managed by Ceph (e.g. those hosts that are listed when you are using vstart).

Reply:
I know but it's the same for the other orchestrator actions on that page. Either hide all actions or use the action behavior we currently have, showing a modal telling you that you need hosts that are managed by the orchestrator.

Reply:
IMO the current behavior/implementation is incorrect. If you select a host that is not managed by Orchestrator and Orchestrator is not activated, then you are able to select the `Delete` button and a dialog pops up telling you that the Orchestrator is not available. This is the first incorrect behavior because such hosts can't be deleted, even if Orchestrator is enabled. After Orchestrator is enabled you can apply the deletion which finally results in an error.

![screenshot](https://user-images.githubusercontent.com/1897962/85376489-126ab300-b538-11ea-95ce-9c2100192ae1.png)


That is the absolute incorrect way a UI should handle this. If it is not possible to perform an action, than the UI should prevent the user from doing so at the first possiblity, and that is by disabling the action button. I will open a tracker issue to fix the `Delete` issue.

Reply:
We discussed that issue in the standup today and come to the conclusion that the UI should prevent the user from doing things that are not possible at the earliest possible stage. Because of that the `Edit` button should be disabled for hosts that are not managed by the Orchestrator. Additionally the action button should use the feature to show a hint why it is disabled in that case.